CONTRIBUTING.md

```diff
@@ -38,6 +40,24 @@ You should also add yourself to the `CONTRIBUTORS.md` file in the root of the pr
   * Then checkout the new branch with `git checkout fix/master/my_contribution`.
   * Please avoid working directly on the `master` branch.
   * You may find the [hub suite of commands](https://github.com/defunkt/hub) helpful
+* Make sure you have added the necessary tests for your changes.
```

Comment:
How about "Make sure you have added sufficient tests and documentation for your changes." to reinforce what's expounded upon below?